### ISSUE TITLE
position, positionX & positionY not working

### ISSUE THREAD

**Issue by philippkuehn** [pixelcog/parallax.js#24]
At the moment custom values for position, positionX & positionY are not working. I've debugged a bit and found out that my settings are always overwrited with "center".

```
$('.foo').parallax({
    imageSrc: 'image.jpg',
    position: 'left bottom', // not working
    positionY: 'bottom', // not working
    positionX: 'left' // not working
});
```

**Comment by mhmli** [comment]
Same issue here too. (Using v1.3 and camel-cased element attribute `data-position-y`.)

**Comment by mikegreiling** [comment]
They appear to be working for me.  Could you provide an example page demonstrating this issue?

Keep in mind that setting `position: 'bottom'` does not necessarily mean the image will always be aligned exactly at the bottom of the parallax container element.  It needs enough extra padding in order to slide up and down as the user scrolls up and down the page.  The "bottom" attribute only comes into account when the viewport is such that there is more vertical "slack" than is needed to achieve the parallax, and it will then align it as close to the "bottom" or "top" as it can while still maintaining the space it needs.

**Comment by mhmli** [comment]
There's a test page at http://stuff.mhm.li/parallaxjs/ for the time being. The lower image is how the parallax image should be positioned before scrolling.

**Comment by mikegreiling** [comment]
Thanks for the information.  I believe I've fixed this issue in v1.3.1.  Try it out and let me know if you have any more issues.

**Comment by mhmli** [comment]
Perfect, thanks for the swift response, Mike.
